core/EE_Dependency_Map.core.php

```diff
@@ -802,16 +802,29 @@ protected function _register_core_dependencies()
                 'EventEspresso\core\services\form\meta\inputs\Select'   => EE_Dependency_Map::load_from_cache,
                 'EventEspresso\core\services\form\meta\inputs\Text'     => EE_Dependency_Map::load_from_cache,
             ],
-            'EventEspresso\core\domain\services\registration\form\LegacyRegistrationForm' => [
+            'EventEspresso\core\domain\services\registration\form\v1\RegForm' => [
```

Comment:
in case these files need to be replaced again in the future...
avoids stupid class names like `NewLegacyRegistrationForm`